### PR TITLE
GH-Issue-54441: Update Pipelines 1.8 tutorial

### PR DESCRIPTION
**Pull request by bobfuru**
xref: PR-50363 - updates IMAGE path in 2 additional places
@openshift/team-documentation PTAL

(https://github.com/openshift/openshift-docs/issues/54441)

### CODE REVIEW

**Comment by bobfuru** [comment]
@openshift/team-documentation PTAL, thanks!

File: modules/op-running-a-pipeline.adoc [hunk 1]
```diff
@@ -42,7 +42,7 @@ $ tkn pipeline start build-and-deploy \
     -w name=shared-workspace,volumeClaimTemplateFile=https://raw.githubusercontent.com/openshift/pipelines-tutorial/{pipelines-ver}/01_pipeline/03_persistent_volume_claim.yaml \
     -p deployment-name=pipelines-vote-ui \
     -p git-url=https://github.com/openshift/pipelines-vote-ui.git \
-    -p IMAGE='image-registry.openshift-image-registry.svc:5000/$(context.pipelineRun.namespace)/pipelines-vote-ui' \
+    -p IMAGE='image-registry.openshift-image-registry.svc:5000/$pipelines-tutorial/pipelines-vote-ui' \
```
Comment (bobfuru) [inline modules/op-running-a-pipeline.adoc:45]: Don't know if this  is also supposed to be updated. It wasn't called out in GH Issue-5441 but assuming it should apply here, too.

Comment (rolfedh) [inline modules/op-running-a-pipeline.adoc:45]:
```suggestion
    -p IMAGE='image-registry.openshift-image-registry.svc:5000/pipelines-tutorial/pipelines-vote-ui' \
```

**Comment by rolfedh** [comment]
Do we need to update `value: image-registry.openshift-image-registry.svc:5000/$(context.pipelineRun.namespace)/$(tt.params.git-repo-name)` in `modules/op-adding-triggers.adoc`, too?

File: modules/op-running-a-pipeline.adoc (hunk 1 above)
Comment (bobfuru) [reply modules/op-running-a-pipeline.adoc:45]: good catch, thanks

**Review by rolfedh — APPROVED** [review]
LGTM

**Comment by rolfedh** [comment]
Nice to see you again, @bobfuru! All the best to you.

**Comment by bobfuru** [comment]
Thanks, @rolfedh - great seeing you as well!

**Pull request merged**